api/models/settings.py

```diff
@@ -0,0 +1,228 @@
+from api.db import db
+
+
+class ApplicationEnvironament:
+
+    def __init__(self):
+        pass
+    DEVELOPMENT = 'development'
+    STAGING = 'staging'
+    PRODUCTION = 'production'
+    TESTING = 'testing'
+
+
+class Settings(db.Model):
+    __tablename__ = 'settings'
+
+    app_environment = db.Column(db.String, default=applicationEnvironament.PRODUCTION)
+    # Name of the application. (Eg. Badgeyay)
+    app_name = db.Column(db.String)
```

Comment:
Will it be of any advantage??

Reply:
Its advantage is by chance, if admin wants to change something he doesn't have to look over the code and rename things, All changes at one place